packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -38,5 +38,7 @@ export const messages = {
   idle_terminated_text:
     'Your debugger session is being terminated because it has been idle for %s minutes.',
   invalid_isv_project_config:
-    'Your project configuration is invalid or incomplete for ISV debugging. Return to the Apex Debugger page in Setup, start a new partner debugging session, and try again.'
+    'Your project configuration is invalid or incomplete for ISV debugging. Return to the Apex Debugger page in Setup, start a new partner debugging session, and try again.',
+  unexpected_error_help_text:
+    'There was an unexpected error launching your debugger session. Please check the debug console for more detailed information.'
```

Comment:
Update as:
An unexpected error occurred while launching the debugger session. Refer to the Output panel for details.